frontend/src/types.ts

```diff
@@ -41,7 +41,7 @@ import { NodeKind } from './queries/schema'
 
 export type Optional<T, K extends string | number | symbol> = Omit<T, K> & { [K in keyof T]?: T[K] }
 
-// Keep this in sync with backend constants (constants.py)
+// Keep this in sync with backend constants/features/{product_name}.yml
```

Comment:
Feels like another thing we could automatically publish from billing so we don't have to manually keep it in sync.